### DISCUSSION TITLE
Feature Request: Add Rider Season Stats Data from PDF to API

### DISCUSSION THREAD

**Discussion by msfintech** [fabricepipart/mgp-timings#291] [Ideas]
Hi Fabrice, I have another feature request that I believe would greatly enhance the functionality of the API.

The source includes a PDF document that provides detailed information on rider progression throughout the race, which is useful for in-depth analysis. The PDF can be found at this link: https://resources.motogp.com/files/results/2024/GER/MotoGP/table1.pdf.

Would it be possible to integrate the data from this PDF into the API? This data would provide valuable insights into the season stats of each rider.

Thank you for considering this request. If you need any further information or assistance, please feel free to contact me.

Best Marco

**Comment by fabricepipart** [comment]
I wonder how feasible that is. Not easy for sure because the framework I use reads the PDF line by line. So it will be hard to know what number corresponds to what column.
I'll give it a try and also look if there is JSON API that would provide the same data.

**Reply by msfintech** [reply]
I understand. If it doesn't work, that's okay. It would be nice, but it's not critical.
